src/main/java/de/adesso/anki/battle/providers/VehicleStateProvider.java

```diff
@@ -5,6 +5,7 @@
 import de.adesso.anki.battle.world.Body;
 import de.adesso.anki.battle.world.World;
 import de.adesso.anki.battle.world.bodies.Mine;
+import de.adesso.anki.battle.world.bodies.Rocket;

```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'de.adesso.anki.battle.world.bodies.Rocket'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/adessoag-github/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.adesso.anki%3Aanki-battle&pullRequest=30&issues=AWRlfcg9l6vpliCe6MRx&open=AWRlfcg9l6vpliCe6MRx)

[ISSUE_KEY]: AWRlfcg9l6vpliCe6MRx